ddl/util/util.go

```diff
@@ -17,6 +17,7 @@ import (
 	"context"
 	"encoding/hex"
 	"fmt"
+	"github.com/pingcap/tidb/sessionctx/variable"
```

Comment:
move this line down to the third part libs

Reply:
done.